scripts/create_newcase

```diff
@@ -82,10 +82,14 @@ def parse_command_line(args, cimeroot):
                         "with the new user grid changes added to it"
 			"This argument is required if --user-grid is True")
 
-    parser.add_argument("--srcroot",default=os.path.dirname(cimeroot),
+    parser.add_argument("--srcroot", default=os.path.dirname(cimeroot),
                         help="Alternative path for source root directory. By default this is set to"
                         "cimeroot/../")
 
+    if cime_model == "acme":
```

Comment:
This is a departure from the normal specification of model options.  If we go down this road we may open the door to any number of component specific features as options to create_newcase.   I think it would be better to make this a cam config option and a compset specification.